deployments/charts/kuma/templates/cp-service.yaml

```diff
@@ -16,20 +16,26 @@ spec:
   ports:
     - port: 5680
       name: diagnostics
+      appProtocol: http
     - port: 5681
       name: http-api-server
+      appProtocol: http
     - port: 5682
       name: https-api-server
+      appProtocol: http
     {{- if ne .Values.controlPlane.environment "universal" }}
     - port: 443
       name: https-admission-server
       targetPort: 5443
+      appProtocol: http
     {{- end }}
     {{- if ne .Values.controlPlane.mode "global" }}
     - port: 5676
       name: mads-server
+      appProtocol: http
     - port: 5678
       name: dp-server
+      appProtocol: grpc
```

Comment:
I don't think this should be gRPC.
Aside from XDS it also serves bootstrap via HTTP.